core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -641,6 +643,28 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  def sendChangeNotifications(workspaceName: WorkspaceName): Future[PerRequestMessage] = {
+
+    val getUsers = {
+      dataSource.inTransaction{ dataAccess =>
+        withWorkspaceContext(workspaceName, dataAccess) {workspaceContext =>
+          requireAccessIgnoreLock(workspaceContext.workspace, WorkspaceAccessLevels.Write, dataAccess) {
+            DBIO.sequence(workspaceContext.workspace.accessLevels.values.map {group =>
+              dataAccess.rawlsGroupQuery.flattenGroupMembership(group)
+            })
+          }
+        }
+      }
+    }
+    getUsers.map { groups =>
+      val userIds = groups.flatten.map(user => user.userSubjectId.value).toSet
+      val notificationMessages = userIds.map { userId => Notifications.WorkspaceChangedNotification(userId, workspaceName) }
+      notificationDAO.fireAndForgetNotifications(notificationMessages)
+      RequestComplete(StatusCodes.OK, userIds)
```

Comment:
Hmmm. Ok I had added that in to make testing (thoroughly) easier... A count could work. I can refactor towards that...